pyaedt/common_rpc.py

```diff
@@ -376,8 +385,8 @@ def launch_ironpython_server(
 
     Parameters
     ----------
-    aedt_path : str
-        AEDT path on Linux.
+    aedt_path : str, optional
+        AEDT path on Linux. If not provided an ANSYSEM_ROOT2xx has to be setup.
     non_graphical : bool, optional
         Whether to start AEDT in non-graphical mode. The default is ``False``, in which case
         AEDT is started in graphical mode.
```

Comment:
```suggestion
    non_graphical : bool, optional
        Whether to start AEDT in non-graphical mode. The default is ``False``, in which case
        AEDT is started in graphical mode.
    port : int, optional
        Port on which the rpc server is to listen. The default is ``18000``.
    launch_client : bool, optional
        Whether to launch the client. The default is ``True``.
    use_aedt_relative_path : bool, optional
       Whether to use the relative path to AEDT. The default is ``False``.
        

```

Reply:
The added parameters must be reviewed for accuracy. The last one might need an explanation of the behavior that occurs when the default of False is used. The format would be: The default is False, in which case the explicit path is used. (I have no idea what the behavior would actually be.)